xml/System.Windows.Forms.DataVisualization.Charting/ChartGraphics.xml

```diff
@@ -20,7 +20,7 @@
   
  Note also that, by default, <xref:System.Drawing.Graphics> functions usually take absolute coordinates. Therefore the conversion methods of this class should be useful if you perform drawing operations.  
   
- A <xref:System.Windows.Forms.DataVisualization.Charting.ChartGraphics> object is exposed in the <xref:System.Windows.Forms.DataVisualization.Charting.ChartPaintEventArgs.ChartGraphics%2A> property of the <xref:System.Windows.Forms.DataVisualization.Charting.ChartPaintEventArgs> class?qualifyHint=False&autoUpgrade=True, which makes it easy to perform custom drawing in the <xref:System.Windows.Forms.DataVisualization.Charting.Chart.PrePaint> and <xref:System.Windows.Forms.DataVisualization.Charting.Chart.PostPaint> events.  
+ A <xref:System.Windows.Forms.DataVisualization.Charting.ChartGraphics> object is exposed in the <xref:System.Windows.Forms.DataVisualization.Charting.ChartPaintEventArgs.ChartGraphics%2A> property of the <xref:System.Windows.Forms.DataVisualization.Charting.ChartPaintEventArgs%2A> class, which makes it easy to perform custom drawing in the <xref:System.Windows.Forms.DataVisualization.Charting.Chart.PrePaint> and <xref:System.Windows.Forms.DataVisualization.Charting.Chart.PostPaint> events.  
```

Comment:
The link to ChartPaintEventArgs is broken; the %2A should be removed, since it shouldn't be used with a type.